Linking against system version of KLT fails on Fedora 31

See here: https://discuss.pixls.us/t/problems-building-current-git-on-fedora-31

To reproduce:
1. Clean building environment, clean clone of `dev`
2. ```klt-devel-1.3.4``` installed (which is the latest version as of writing)
3. Build RT using ```-DWITH_SYSTEM_KLT="ON"```
4. Linking fails with ```undefined reference to `KLTCreateTrackingContext'``` or others.

Can anyone test on different Linux distro's?

I confirm that today `dev` branch fails to build using system KLT:
```
/usr/bin/ld: ../rtengine/librtengine.a(calc_distort.cc.o): in function `calcDistortion(unsigned char*, unsigned char*, int, int, int, double&)':
/home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:48: undefined reference to `KLTCreateTrackingContext'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:55: undefined reference to `KLTCreateFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:56: undefined reference to `KLTCreateFeatureTable'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:62: undefined reference to `KLTSelectGoodFeatures'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:63: undefined reference to `KLTStoreFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:65: undefined reference to `KLTTrackFeatures'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:66: undefined reference to `KLTStoreFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:279: undefined reference to `KLTFreeFeatureTable'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:280: undefined reference to `KLTFreeFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:281: undefined reference to `KLTFreeTrackingContext'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:160: undefined reference to `KLTFreeFeatureTable'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:161: undefined reference to `KLTFreeFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:162: undefined reference to `KLTFreeTrackingContext'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:270: undefined reference to `KLTFreeFeatureTable'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:271: undefined reference to `KLTFreeFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:272: undefined reference to `KLTFreeTrackingContext'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:261: undefined reference to `KLTFreeFeatureTable'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:262: undefined reference to `KLTFreeFeatureList'
/usr/bin/ld: /home/rpmbuild/rpmbuild/BUILD/RawTherapee-dev/rtengine/calc_distort.cc:263: undefined reference to `KLTFreeTrackingContext'
collect2: error: ld returned 1 exit status
```

I see that `calc_distort.cc` has not been changed lately, but maybe these two lines:
https://github.com/Beep6581/RawTherapee/blob/7b54c7f2a42da28f38bdaf5399c0ef3e339e58dc/rtengine/calc_distort.cc#L8-L9
should not statically refer to the bundled KLT?

--- a/rtengine/CMakeLists.txt
+++ b/rtengine/CMakeLists.txt
@@ -12,6 +12,11 @@ include_directories(${EXTRA_INCDIR}
     ${LENSFUN_INCLUDE_DIRS}
     ${RSVG_INCLUDE_DIRS}
 )
+if(NOT WITH_SYSTEM_KLT)
+    include_directories("${CMAKE_SOURCE_DIR}/rtengine/klt")
+else()
+    include_directories(${KLT_INCLUDE_DIRS})
+endif()
 
 link_directories("${PROJECT_SOURCE_DIR}/rtexif"
     ${EXPAT_LIBRARY_DIRS}
@@ -184,6 +189,7 @@ target_link_libraries(rtengine rtexif
     ${ZLIB_LIBRARIES}
     ${LENSFUN_LIBRARIES}
     ${RSVG_LIBRARIES}
+    ${KLT_LIBRARIES}
 )
 
 install(FILES ${CAMCONSTSFILE} DESTINATION "${DATADIR}" PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ)

--- a/rtengine/calc_distort.cc
+++ b/rtengine/calc_distort.cc
@@ -5,8 +5,8 @@ locations (before and after tracking) to text files and to PPM files,
 and prints the features to the screen.
 **********************************************************************/
 
-#include "klt/pnmio.h"
-#include "klt/klt.h"
+#include <pnmio.h>
+#include <klt.h>
 #include <cmath>
 #include <cstring>
 

